app/webpacker/components/wca/FormBuilder/input/FormInputs.js

```diff
@@ -196,7 +196,7 @@ export const InputTextArea = wrapInput((props) => (
 
 export const InputNumber = wrapInput((props) => {
   const onChangeNumber = useCallback((e, { value: newValue }) => {
-    const convertedNumber = Number(newValue);
+    const convertedNumber = (props.nullable && newValue === '') ? '' : Number(newValue);
```

Comment:
This is essentially correct, but very dense. Can you split the logic into two or even three lines perhaps?

Reply:
Updated!